tests/ui/test_ui_tools.py

```diff
@@ -1908,61 +1909,76 @@ def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture, widget_size,
             write_box.msg_write_box.set_edit_text.assert_not_called()
 
     @pytest.mark.parametrize('raw_html, expected_content', [
-        ("""<blockquote>
-                <p>A</p>
-            </blockquote>
-            <p>B</p>""", "{} A\n\nB"),
-        ("""<blockquote>
-                <blockquote>
+        case("""<blockquote>
                     <p>A</p>
                 </blockquote>
-                <p>B</p>
-            </blockquote>
-            <p>C</p>""", "{} {} A\n\n{} B\n\nC"),
-        ("""<blockquote>
-                <blockquote>
+                <p>B</p>""",
+             ("{} A\n\n"
+              "B"),
+             id="quoted level 1"),
+        case("""<blockquote>
                     <blockquote>
                         <p>A</p>
                     </blockquote>
                     <p>B</p>
                 </blockquote>
-                <p>C</p>
-            </blockquote>
-            <p>D</p>""", "{} {} {} A\n\n{} {} B\n\n{} C\n\nD"),
-        ("""<blockquote>
-                <p>A<br/>B</p>
-            </blockquote>
-            <p>C</p>""", "{} A\n{} B\n\nC"),
-        ("""<blockquote>
-                <p><a href='https://chat.zulip.org/'</a>czo</p>
-            </blockquote>""", "{} czo [1]\n"),
-        pytest.param("""<blockquote>
-                            <blockquote>
-                                <p>A<br>
-                                B</p>
-                            </blockquote>
-                        </blockquote>
-            """, "{} {} A\n{} {} B",
-                     marks=pytest.mark.xfail(reason="rendered_bug")),
-        pytest.param("""<blockquote>
-                            <blockquote>
-                                <p>A</p>
-                            </blockquote>
-                            <p>B</p>
-                            <blockquote>
-                                <p>C</p>
-                            </blockquote>
+                <p>C</p>""",
+             ("{} {} A\n\n"
+              "{} B\n\n"
+              "C"),
+             id="quoted level 2"),
+        case("""<blockquote>
+                    <blockquote>
+                        <blockquote>
+                            <p>A</p>
                         </blockquote>
-        """, "{} {} A\n{} B\n{} {} C",
-                     marks=pytest.mark.xfail(reason="rendered_bug")),
-    ], ids=[
-        "quoted level 1",
-        "quoted level 2",
-        "quoted level 3",
-        "multi-line quoting",
-        "quoting with links",
-        "multi-line level 2",
-        "quoted level 2-1-2",
+                        <p>B</p>
+                    </blockquote>
+                    <p>C</p>
+                </blockquote>
+                <p>D</p>""",
+             ("{} {} {} A\n\n"
+              "{} {} B\n\n"
+              "{} C\n\n"
+              "D"),
+             id="quoted level 3"),
+        case("""<blockquote>
+                    <p>A<br>B</p>
```

Comment:
Does the server send this over two lines?

Reply:
Ahh, yes. Sorry, forgot to change this!